client/src/reducers/userFavoritesReducer.js

```diff
@@ -3,21 +3,31 @@ import _ from 'lodash';
 
```

Comment:
Import methods needed from lodash by their names, rather than importing the whole of lodash with an underscore(_) which I don't think is descriptive enough.